src/app/components/notifier/notifier.service.js

```diff
@@ -18,7 +18,7 @@
 
     function info(text) {
       notifications.info("Information", text, {
-        duration: 20000,
+        duration: 5000,
```

Comment:
Yes. That's ok for `MuteNotifier`. This one is regular `Notifier`, where user interaction is not expected.